backend/apps/volontulo/filters.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+
+"""
+.. module:: api
+"""
+
+from rest_framework import filters
+
+
+class IsOfferJoinedFilter(filters.BaseFilterBackend):
+
+    """Filter only to offers that are alredy joined."""
```

Comment:
Docstring does not specify explicitly returned objects - I mean we have also excluded items.